Microblogging/src/app/services/user.service.ts

```diff
@@ -119,8 +119,22 @@ export class UserService {
       lastName: updatedUser.lastName,
       imageURL: updatedUser.imageURL,
       about: updatedUser.about,
-      password: ''
+      password: password
     }
     return this.http.put<User>(`${this.url}/about`, obj, this.httpOptions)
```

Comment:
This function will not set a new password, you need to make a new function that hits the url/password put mapping

Reply:
fixed, thought I deleted it before